### PR TITLE
✨ CONTRAST-32217 Automatic updates with curl example

### PR DESCRIPTION
**Pull request by gilday**
We publish the Contrast Java agent to Maven Central. Hosting the agent on Maven Central enables users to pull down the latest agent using a plethora of dev and ops tools. Some users want to configure their systems to automatically update their Contrast Java agent; this examples shows Linux users how to automatically update the Java agent using a scheduled job and `curl`.

This example uses an Ubuntu 18.04 LTS system. It includes all the steps necessary to install a Contrast Java agent jar on the system, update it automatically every night, configure it to communicate with Contrast UI, and verify that the jar is working.

### CODE REVIEW

File: content/installation/java/package/scheduled-update-with-curl.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+Device: 801h/2049d      Inode: 256034      Links: 1
+Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
+Access: 2019-04-11 02:02:01.265775928 +0000
+Modify: 2019-04-11 02:24:47.849796936 +0000
+Change: 2019-04-11 02:24:47.849796936 +0000
+ Birth: -
+```
+
+## Step 2: Gather Contrast account information 
+
```
Comment (kaitlynmortimer) [inline content/installation/java/package/scheduled-update-with-curl.md]: @gilday I made this its own step to mirror the last Maven article.

File: content/installation/java/package/scheduled-update-with-curl.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+When Contrast is installed on a Linux host, users typically want Contrast-enabled web applications on the host to share basic configuration parameters, such as the ones required to connect to Contrast UI. By convention, Contrast look for configuration in a YAML file at path */etc/contrast/java/contrast_securtiy.yaml* on Linux hosts.
+
+* Create the */etc/contrast/java* directory:
+
+```console
+sudo mkdir -p /etc/contrast/java
+```
+
+* Use `tee` to create the configuration file. Press `CTRL+D` when you've finished typing all the lines. 
+
```
Comment (kaitlynmortimer) [inline content/installation/java/package/scheduled-update-with-curl.md]: @gilday Are they supposed to use CTRL+D before or after adding the info from the UI?

File: content/installation/java/package/scheduled-update-curl.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+When Contrast is installed on a Linux host, users typically want Contrast-enabled web applications on the host to share basic configuration parameters, such as the ones required to connect to Contrast UI. By convention, Contrast look for configuration in a YAML file at path */etc/contrast/java/contrast_securtiy.yaml* on Linux hosts.
+
+* Create the */etc/contrast/java* directory:
+
+```console
+sudo mkdir -p /etc/contrast/java
+```
+
+* Use `tee` to create the configuration file. Press `CTRL+D` when you've finished typing all the lines. 
+
```
Comment (kaitlynmortimer) [inline content/installation/java/package/scheduled-update-curl.md]: Re-commenting: 

@gilday Are they supposed to use CTRL+D before or after adding the info from the UI?

File: content/installation/java/package/scheduled-update-curl.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+Device: 801h/2049d      Inode: 256034      Links: 1
+Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
+Access: 2019-04-11 02:02:01.265775928 +0000
+Modify: 2019-04-11 02:24:47.849796936 +0000
+Change: 2019-04-11 02:24:47.849796936 +0000
+ Birth: -
+```
+
+## Step 2: Gather Contrast account information 
+
```
Comment (kaitlynmortimer) [inline content/installation/java/package/scheduled-update-curl.md]: Re-commenting: 

@gilday I made this its own step to mirror the last Maven article.

File: content/installation/java/package/scheduled-update-curl.md (hunk 3 above)
Comment (gilday) [reply content/installation/java/package/scheduled-update-curl.md]: After type all the info into the console, they press `CTRL+D` to write the information to the file and exist the `tee` program

File: content/installation/java/package/scheduled-update-curl.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+```console
+vagrant up
+```
+
+```console
+vagrant ssh
+```
+
```
Comment (kaitlynmortimer) [inline content/installation/java/package/scheduled-update-curl.md]: @gilday Are there any prerequisites?

File: content/installation/java/package/scheduled-update-with-curl.md (hunk 2 above)
Comment (gilday) [reply content/installation/java/package/scheduled-update-with-curl.md]: After type all the info into the console, they press CTRL+D to write the information to the file and exist the `tee` program

File: content/installation/java/package/scheduled-update-curl.md (hunk 4 above)
Comment (gilday) [reply content/installation/java/package/scheduled-update-curl.md]: ok

File: content/installation/java/package/scheduled-update-curl.md (hunk 5 above)
Comment (gilday) [reply content/installation/java/package/scheduled-update-curl.md]: Yes, install `vagrant`

Comment (gilday) [reply content/installation/java/package/scheduled-update-curl.md]: I put a link to https://www.vagrantup.com/

**Comment by gilday** [comment]
@kaitlynmortimer we used this `tee` method of creating a new file from the terminal in the Gradle article as well. It's not a normal way to create files, but @rszewczyk felt it was the most copy / paste friendly way for a terminal newbie who is not familiar with relatively complicated CLI editors like `vim` and `emacs`

**Comment by tylernhoward** [comment]
I will do this too from scratch to make sure we aren't missing any dependencies or anything

File: content/installation/java/package/scheduled-updates-curl.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+## About Updates with Maven Central 
+
+Some users like to automatically update their Contrast Java agent software to the latest version. Linux users can schedule Java agent updates from Maven Central using common Linux tools `cron` and `curl`. The following instructions show you how to configure a scheduled Java agent job on an Ubuntu 18.04 Linux host.
+
+> **Note:** Use your preferred editor to create a file with the following contents. The examples provided use `tee` to create the file. 
+
+## Before You Start
+
+If you want to perform each step as you follow along with this guide, you can use [Vagrant](https://www.vagrantup.com/) to create a new Ubuntu 18.04 virtual machine:
```
Comment (tylernhoward) [inline content/installation/java/package/scheduled-updates-curl.md]: We should also instruct users to download VirtualBox here: https://www.virtualbox.org/wiki/Downloads

Comment (tylernhoward) [reply content/installation/java/package/scheduled-updates-curl.md]: >>> If you want to perform each step as you follow along with this guide, you can use [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/wiki/Downloads) to create a new Ubuntu 18.04 virtual machine:

File: content/installation/java/package/scheduled-update-curl.md (hunk 5 above)
Comment (gilday) [reply content/installation/java/package/scheduled-update-curl.md]: I am going to add a prerequisite based on Tyler's feedback

File: content/installation/java/package/scheduled-updates-curl.md (hunk 6 above)
Comment (gilday) [reply content/installation/java/package/scheduled-updates-curl.md]: Good catch, added

**Comment by gilday** [comment]
@kaitlynmortimer I just verified that this works and this is now ready to ship

**Pull request merged**